csuntechlab/iSTART
# STR-218 -- Implement multi-question quiz template

# Description
Implemented functionality to support cases for multi question quiz.
  
# Testing
1. Visit `index 30` to view the MultiQuizInput template.
2. Test for toggle of inputs per question.
3. Ensure continue button is enabled only when all question filled out.
4. On `index 31` ensure the proper response appears minus the `or` and `all` case.
**This has object values to change. If you want to test special cases, they are referred to in object by `andAll` & `orAll` along with `caseResponse` for the response of special cases.**
